cli/args/flags.rs

```diff
@@ -5079,17 +5079,21 @@ fn watch_arg_parse_with_paths(
     });
   }
 
-  matches
-    .remove_many::<String>("hmr")
-    .map(|paths| WatchFlagsWithPaths {
-      paths: paths.collect(),
-      hmr: true,
-      no_clear_screen: matches.get_flag("no-clear-screen"),
-      exclude: matches
-        .remove_many::<String>("watch-exclude")
-        .map(|f| f.collect::<Vec<String>>())
-        .unwrap_or_default(),
-    })
+  if matches.try_contains_id("hmr").is_ok() {
```

Comment:
Is this condition necessary?

Reply:
Yes, the test command doesn't support hmr, which results in panic if you try to get the argument
![image](https://github.com/user-attachments/assets/6afbf762-69b6-43b1-b538-200d466c1f7a)
![image](https://github.com/user-attachments/assets/e17ce453-cb9f-4424-aaa3-27a18e5dc257)